snapshot-labs/sx-monorepo
# fix(ui): fully round button on loading state

### Summary



Make button in `loading` state fully round

Before fix

![Screenshot 2024-09-17 at 11 28 32](https://github.com/user-attachments/assets/e198612a-cbe0-4dc9-b963-eef16dd7b235)

After fix

![Screenshot 2024-09-17 at 11 28 45](https://github.com/user-attachments/assets/f923fb41-e1d2-4efe-93da-6b3378b4cb5b)

This will fix the PUBLISH button in editor page, and the FOLLOW SPACE button
Also moved the `!px-0 w-[46px]` classes inside button component

### How to test

1. Go to a proposal creation page
2. The Publish button should be fully rounded when on loading state
3. Follow/Unfollow a space
4. The button should be fully rounded when on loading state
